spec/index.html

```diff
@@ -248,6 +248,31 @@ <h2>Example</h2>
   <p><emu-xref aoid="Get"></emu-xref> is an abstract operation from ES6</p>
 </emu-clause>
 
+<emu-clause id="emu-not-ref" namespace="emu-not-ref">
+  <h1>emu-not-ref</h1>
+  <p>Suppresses automatic linking to definitions.</p>
+
+  <h2>Example</h2>
+  <pre><code class="language-html">
+  &lt;div id="not-ref-section-1">
+    &lt;p>An &lt;dfn>example&lt;/dfn> is used for illustrative purposes.&lt;/p>
+  &lt;/div>
+  &lt;div id="not-ref-section-2">
+    &lt;p>When a word defined in another section (or algorithm step) is used, as in this example, it is normally automatically linked to the section containing the definition.&lt;/p>
+    &lt;p>When such a word should not be automatically linked, for &lt;emu-not-ref>example&lt;/emu-not-ref> when using its colloquial definition, it can be wrapped with this tag to surpress the automatic linking.&lt;/p>
+  &lt;/div>
+  </code></pre>
+
+  <b>Result</b>
+  <div id="not-ref-section-1">
+    <p>An <dfn>example</dfn> is used for illustrative purposes.</p>
+  </div>
+  <div id="not-ref-section-2">
+    <p>When a word defined in another section (or algorithm step) is used, as in this example, it is normally automatically linked to the section containing the definition.</p>
```

Comment:
Shouldn't the word "example" be auto-linked here?

Reply:
It will be once this gets rendered! This document is written in ecmarkup.

Reply:
:-p too bad the extension implies it's HTML